sdk/servicebus/service-bus/src/models.ts

```diff
@@ -23,6 +23,35 @@ export interface MessageHandlers<ReceivedMessageT> {
   processError(err: Error): Promise<void>;
 }
 
+/**
+ * Represents all the receive modes for the receiver.
+ *
+ * @type
+ */
+export type ReceiveModes = "peekLock" | "receiveAndDelete";
+
+/**
+ *
+ *
+ * @interface CreateReceiverOptions
+ * @template ReceiveModeT
+ */
+export interface CreateReceiverOptions<ReceiveModeT extends ReceiveModes> {
+  /**
+   * Represents the receive mode for the receiver.
+   *
+   * In receiveAndDelete mode, messages are deleted from Service Bus as they are received.
+   *
+   * In peekLock mode, messages are not deleted from the queue directly, you need to settle the message by calling complete(), abandon() or defer() methods on
+   * the message.
```

Comment:
Let's use similar text as that in the docs for `createReceiver()`

```suggestion
   * In peekLock mode, the receiver has a lock on the message for the duration specified on the
   * queue/subscription.
   *
   * Messages that are not settled within the lock duration will be redelivered as many times as
   * the max delivery count set on the queue/subscription, after which they get sent to a separate 
   * dead letter queue.
   * 
   * You can settle a message by calling complete(), abandon(), defer() or deadletter() methods on
   * the message.
```